modules/ROOT/pages/fd-scatter-gather.adoc

```diff
@@ -0,0 +1,259 @@
+= Run Different Processes in Parallel, then Gather the Results
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+With a Scatter-Gather card, you can send a message through more than one set of cards at a time, collect the results from each set, and pass those results (along with the original message) to the next card in a flow.
+
+.This schematic diagram shows a Scatter-Gather card that includes the minimum of two routes.
+image::fd-sg.png[A Scatter-Gather card in a flow]
+<1> Any card can pass messages to a Scatter-Gather card, and this card can be the trigger of the current flow or any other card in the flow.
```

Comment:
It cannot be trigger!

Reply:
@matibenav , the paragraph doesn't say that it can be a trigger. But I'll make the wording more clear.